web/src/search/input/QueryInput.tsx

```diff
@@ -396,7 +398,7 @@ export class QueryInput extends React.Component<Props, State> {
                                         {this.state.loadingSuggestions && (
                                             <li className="suggestion suggestion--selected">
                                                 <LoadingSpinner className="icon-inline" />
-                                                <div className="suggestion__description">fetching suggestions...</div>
+                                                <div className="suggestion__description">LOADING</div>
```

Comment:
Sorry - when I said capitalized, I meant "Loading" 😅 
```suggestion
                                                <div className="suggestion__description">Loading</div>
```

Reply:
Ah, I understood uppercase. Fixed :+1: